website/versions.json

```diff
@@ -1,6 +1 @@
-[
-  "2.0.0-alpha.41",
-  "2.0.0-alpha.40",
-  "2.0.0-alpha.39",
-  "2.0.0-alpha.38"
-]
+["2.0.0-alpha.42", "2.0.0-alpha.40", "2.0.0-alpha.39", "2.0.0-alpha.38"]
```

Comment:
Can we change it back to how it was? Easier to see diffs to this file in future.